gulpfile.js

```diff
@@ -143,17 +134,10 @@ gulp.task('tests', function(done) {
 
 
 
-
-
-
-
-
-
-
 // task chains
-gulp.task('default', ['styles', 'fonts', 'views', 'css-vendors', 'js-vendors', 'scripts', 'watch']); // build app
+gulp.task('default', ['styles', 'fonts', 'views', 'js-vendors', 'scripts', 'watch']); // build app
 gulp.task('serve', ['styles', 'scripts', 'fonts', 'views', 'browser-sync-server', 'watch']); // launch server
 gulp.task('build', ['styles', 'scripts', 'fonts', 'views', 'browser-sync', 'watch']); // proxy
-gulp.task('vendors', ['css-vendors', 'js-vendors']); // vendor js and css
+gulp.task('vendors', ['js-vendors']); // vendor js and css
 gulp.task('test', ['tests', 'watch']);
 gulp.task('test-once', ['tests']);
```

Comment:
We might want to drop all these changes. I made a lot of changes to the gulpfile in #12. For the most part I've covered your changes here, plus many more. At the very least, it's going to cause merge conflicts.